### PR TITLE
docs: proper setup to enable "Exclude Objects"

### PR DESCRIPTION
**Pull request by valadas**
Documents how to properly setup to enable "Exclude Objects"

Signed-off-by: Daniel Valadas info@danielvaladas.com

### CODE REVIEW

**Review by matmen — COMMENTED** [review]
Hey, thanks for your PR! I knew I forgot something.. 😅 Looks good to me so far, just one small suggestion:

File: docs/features/gcode_viewer.md [hunk 1]
```diff
@@ -33,6 +33,13 @@ icon in the header of the card.
 Excluded objects will be marked in red, the currently printing object in blue, and all
 other objects in green.
 
+For this feature to work you need to:
+- Turn on `Label Objects` in your slicer.
+- Add an `[exclude_object]` section to your printer.cfg or fluidd.cfg file.
```
[the remark this answers is not in the document]
Comment (pedrolamas) [reply docs/features/gcode_viewer.md:38]: The fluidd.cfg file contains just the "basics" required by Fluidd to work and nothing else, so I don't think we should add anything else, regardless of impact (if not, why not also add `gcode_arcs`, `sdcard_loop`, ... ?)

Let's leave it out of the basics! 🙂

**Comment by pedrolamas** [comment]
Thank you for your contribution @valadas.

However, we have [contribution guidelines](https://github.com/fluidd-core/fluidd/blob/develop/CONTRIBUTING.md) that must be followed, specifically "Commit messages should follow the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard, and should have a Signed-off-by line"

To make things easier, I would ask that you edit your first post on this PR and just add the `Signed-off-by: Your Name <your email address>` footer.

File: docs/features/gcode_viewer.md [hunk 2]
```diff
@@ -33,6 +33,14 @@ icon in the header of the card.
...
+- Add a a line `enable_object_processing: True` to your mooraker.conf `[file_manager]` section.
```
Comment (pedrolamas) [inline docs/features/gcode_viewer.md:39]:
```suggestion
- Add `enable_object_processing: True` to the `[file_manager]` section in your moonraker.conf file.
```

**Review by pedrolamas — CHANGES_REQUESTED** [review]
Fixed a typo (double "a") and clarified a bit the text.

**Review by matmen — APPROVED** [review]
LGTM

**Review by pedrolamas — APPROVED** [review]

**Pull request merged**